### PR TITLE
feat: add tests for ListGroup and other linter issues

### PR DESCRIPTION
**Pull request by suwarnoong**
Tests for https://github.com/SAP/fundamental-vue/issues/200

### CODE REVIEW

File: ui/src/components/ListGroup/__tests__/ListGroup.test.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      }
+    }
+  });
+
+  it("renders correctly", () => {
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it("has correct number of list items", () => {
+    expect(wrapper.findAll("li").length).toBe(2);
```
Comment (ChristianKienle) [inline ui/src/components/ListGroup/__tests__/ListGroup.test.ts]: I think when you use:

```javascript
expect(wrapper.findAll("li")).toHaveLength(2);
```

you will test the same thing but jest will give you a better error message if the test fails.

File: ui/src/components/ListGroup/__tests__/ListGroupItem.test.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      }
+    }
+  });
+
+  it("renders correctly", () => {
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it("has correct number of actions", () => {
+    expect(wrapper.findAll(".action-element").length).toBe(2);
```
Comment (ChristianKienle) [inline ui/src/components/ListGroup/__tests__/ListGroupItem.test.ts]: I think when you use:

```javascript
expect(wrapper.findAll(".action-element")).toHaveLength(2);
```

you will test the same thing but jest will give you a better error message if the test fails.

File: ui/src/components/ListGroup/__tests__/ListGroup.test.ts (hunk 1 above)
Comment (suwarnoong) [reply ui/src/components/ListGroup/__tests__/ListGroup.test.ts]: got it. changing now.

**Comment by suwarnoong** [comment]
Oh... this supposed to be a test branch. Will do it.

**Pull request closed without merging**